src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +94,10 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate todayDate = LocalDate.now();
+        return someDate.toString() + (someDate.equals(todayDate) ? " is today"
+                        : someDate.isAfter(todayDate) ? " is after " + todayDate.toString()
+                        : " is before " + todayDate.toString());
```

Comment:
let's check isAfter() isBefore() if both false than consider is today without checking